highlight/highlight
# Improve trace display on network resource panel

## Summary

Moves the flame graph inside the `Trace` tab on a network resource.

<img width="543" alt="Screenshot 2024-01-18 at 12 55 12 PM" src="https://github.com/highlight/highlight/assets/308182/74073b5a-984d-4a38-863e-3f693b3da7c4">

Also adds a loading and empty state on a trace that explains why the data might not be available.

![image](https://github.com/highlight/highlight/assets/308182/4285652f-f174-487c-b11c-4c0d4ea6f366)

## How did you test this change?

Local + PR preview click test using the following steps:

* View a session
* Go to network resources
* View all types of resources and ensure the `Trace` tab is shown correctly
* View the trace tab and ensure the UI works as expected

## Are there any deployment considerations?

N/A - client only changes

## Does this work require review from our design team?

Will run these screenshots by @julian-highlight!
